src/server/database/Database/DatabaseWorkerPool.cpp

```diff
@@ -40,8 +40,8 @@
 #endif
 
 #if MARIADB_VERSION_ID >= 100600
-#define MIN_MYSQL_SERVER_VERSION 100200u
-#define MIN_MYSQL_CLIENT_VERSION 30203u
+#define MIN_MYSQL_SERVER_VERSION 100500u
+#define MIN_MYSQL_CLIENT_VERSION 30503u
```

Comment:
I'm not sure about this line.

Reply:
me neither, but I could not find where it is located

Reply:
in `MariaDB 10.6` only

Reply:
then I will just change back client thing? Or do you have time to test it fast with your setup?

I only run mysql